dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/AccessLogFilter.java

```diff
@@ -67,139 +67,143 @@
 
     private static final String ACCESS_LOG_KEY = "dubbo.accesslog";
 
-    private static final String FILE_DATE_FORMAT = "yyyyMMdd";
-
-    private static final String MESSAGE_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
-
     private static final int LOG_MAX_BUFFER = 5000;
 
     private static final long LOG_OUTPUT_INTERVAL = 5000;
 
-    private final ConcurrentMap<String, Set<String>> logQueue = new ConcurrentHashMap<String, Set<String>>();
+    private static final SimpleDateFormat FILE_DATE_FORMATTER = new SimpleDateFormat("yyyyMMdd");
 
-    private final ScheduledExecutorService logScheduled = Executors.newScheduledThreadPool(2, new NamedThreadFactory("Dubbo-Access-Log", true));
+    private static final SimpleDateFormat MESSAGE_DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 
-    private volatile ScheduledFuture<?> logFuture = null;
+    private final ConcurrentMap<String, Set<AccessLogData>> logQueue = new ConcurrentHashMap<String, Set<AccessLogData>>();
```

Comment:
Can this field be static?

Reply:
I think it make sense now to make it static as it is now bound to single thread. 

In addition to this, I was thinking (correct me If I am wrong), instead of defining the logQueue as a member of this class, we can it get through a factory (although factory will supply single queue only), so that in future even if we have change it based on our defined we don't have to touch this class. May be we can do this as part of separate PR. What do you say?

Reply:
emmm...
What is this factory produce? Maybe I haven't get your point~ Can u give more details?

Reply:
it provide the storage object which its consumer object, 

e.g.   
   ```
Store store= InMemoryQueueStorageFactory.getInstance().getStore();
   store.add(accessLog);
```

something like above. The above example what I hvae given may not be very intuitive itself for explaination, but  for high level understanding it might help. If you want I might create a sample branch to demonestrate in more detail way by having some implementation. What do you say?

Reply:
Personally think that it is not necessary, because it will make the code more complicated. I think that the current structure is relatively simple, haha.

Reply:
😄